dags/hivemind_etl_helpers/discord_mongo_vector_store_etl.py

```diff
@@ -47,7 +47,7 @@ def process_discord_guild_mongo(community_id: str) -> None:
         from_date += timedelta(seconds=1)
 
     documents = discord_raw_to_docuemnts(guild_id=guild_id, from_date=from_date)
-    node_parser = configure_node_parser(chunk_size=256)
+    node_parser = configure_node_parser(chunk_size=512)
```

Comment:
I think either 256 or 512 would work. I wonder though why you made this change here

Reply:
It was because while I was extracting 1 month of RnDAO embeddings, in some cases having long metadata would break to ETL process (llama_index throwing errors), so I changed it to 512.

Reply:
Alright, thanks for clarifying